src/content/guides/scaffolding.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Scaffolding
+contributors:
+  - evenstensberg
+  - pranshuchittora
+sort: 2
+---
+
+It can be hard to set up a complex webpack configuration for the first time. Writing advanced configurations to optimize performance is quite hard. The `init` feature is designed to support people that want to create their own configuration or initializing projects that other people create.
+
+## Creating a scaffold
+
+Before writing a `webpack-cli` scaffold, think about what you're trying to achieve. Do you want a "general" scaffold that could be used by any project or type of app? Do you want something focused, like a scaffold that writes both your `webpack.config.js` and your framework code? It's also useful to think about the user experience for your scaffold.
+
+`webpack-cli` offers an interactive experience to customize the output accordingly. For example asking questions like: "What is your entry point?".
+
+### Writing a scaffold
+
+There are many resources where you can learn how to write a scaffold, you can start from [Writing a Scaffold](/contribute/writing-a-scaffold/)
+
+`webpack-scaffold` is a utility suite for creating scaffolds. It contains functions that could be used to create a scaffold.
+
+### Running a scaffold
+
+A scaffold can be executed using `webpack-cli init`:
+
+```bash
+webpack-cli init <your-scaffold>
+```
+
+#### Running a scaffold locally
+
+When the scaffold package is in your local file system you should point `init` to its path:
+
+```bash
+webpack-cli init path/to/your/scaffold
+```
+
+Or you can create a global module and symlink to the local one:
+
+- Using npm
+  `bash cd path/to/my-scaffold npm link webpack-cli init my-scaffold`
```

Comment:
<img width="742" alt="Screen Shot 2019-03-25 at 11 25 30 AM" src="https://user-images.githubusercontent.com/10549495/54908501-b9edd580-4ef0-11e9-9518-768ae39dede3.png">
Does it look correct for you when the page is on website?

Reply:
@pranshuchittora this doesnt look right

Reply:
Fixed
TIp:There must be an extra blank line surround the list items. 
```
./src/content/guides/scaffolding.md: 41: MD032/blanks-around-lists Lists should be surrounded by blank lines [Context: "```bash"]
./src/content/guides/scaffolding.md: 42: MD031/blanks-around-fences Fenced code blocks should be surrounded by blank lines [Context: "```bash"]
./src/content/guides/scaffolding.md: 47: MD032/blanks-around-lists Lists should be surrounded by blank lines [Context: "```bash"]
./src/content/guides/scaffolding.md: 48: MD031/blanks-around-fences Fenced code blocks should be surrounded by blank lines [Context: "```bash"]
```